.build/flags.mk

```diff
@@ -4,7 +4,8 @@ LIST_PKGS ?= $(shell go list ./... | grep -v /vendor/)
 
 # Many Go tools take file globs or directories as arguments instead of packages.
 ROOT_PKG_FILES := $(wildcard *.go)
-PKG_NAMES := auth config dig internal metrics modules service testutils tracing ulog
+# Get all the go packages, ignore vendor and take fx/* directories
+PKG_NAMES := $(shell go list ./... | grep -v "vendor\|examples" |  cut -d"/" -f 3 | uniq | paste -sd " " -)
```

Comment:
The grep -v seems a bit fragile. Can we do something like `$(shell go list $(glide nv) | grep -v /examples` in case we have some other examples package that we do want to test?